src/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -106,9 +106,9 @@ type ExecSync = (command: string, options: ExecSyncOptions) => string;
 export class NodeTypingsInstaller extends TypingsInstaller {
     private readonly nodeExecSync: ExecSync;
     private readonly npmPath: string;
-    readonly typesRegistry: Map<string, MapLike<string>>;
+    typesRegistry: Map<string, MapLike<string>> = undefined!;

```

Comment:
Does this need `undefined!`? It appears to be assigned in the constructor (otherwise I would personally prefer an assertDefined instead)

Reply:
oops thats from last change. it doesnt need to change ..